### PR TITLE
New 65 ANSI Blocker layouts: Split BS, Tsangan

### PR DESCRIPTION
**Pull request by brandonschlack**
Adds 2 65 ANSI Blocker layout variations as default layouts: Split BS & Tsangan. Since many 65 ANSI Blocker keyboards also support support split backspace and 7u bottom row configurations, adding these default layouts will enable community layout support for these popular configurations going forward.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* N/A

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: layouts/default/65_ansi_blocker_tsangan/default_65_ansi_blocker/keymap.c [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+     * │ Tab │ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │ [ │ ] │  \  │PgU│
+     * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┼───┤
+     * │ Caps │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ; │ ' │  Enter │PgD│
+     * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────┬───┼───┤
+     * │ Shift  │ Z │ X │ C │ V │ B │ N │ M │ , │ . │ / │ Shift│ ↑ │End│
+     * ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬─┬───┼───┼───┤
+     * │Ctrl│GUI │Alt │                        │ Alt│Ctrl│ │ ← │ ↓ │ → │
+     * └────┴────┴────┴────────────────────────┴────┴────┘ └───┴───┴───┘
+     */
+    [0] = LAYOUT_65_ansi_blocker(
```
Comment (noroadsleft) [inline layouts/default/65_ansi_blocker_tsangan/default_65_ansi_blocker/keymap.c:17]:
```suggestion
    [0] = LAYOUT_65_ansi_blocker_tsangan(
```

File: layouts/default/65_ansi_blocker_tsangan/default_65_ansi_blocker/keymap.c [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_HOME,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_PGUP,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,  KC_PGDN,
+        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_END,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                                      KC_RALT, KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
```
Comment (noroadsleft) [inline layouts/default/65_ansi_blocker_tsangan/default_65_ansi_blocker/keymap.c:22]:
```suggestion
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                                               KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
```

File: layouts/default/65_ansi_blocker_tsangan/info.json [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                {"x":5, "y":0},
+                {"x":6, "y":0},
+                {"x":7, "y":0},
+                {"x":8, "y":0},
+                {"x":9, "y":0},
+                {"x":10, "y":0},
+                {"x":11, "y":0},
+                {"x":12, "y":0},
+                {"x":13, "y":0},
+                {"x":14, "y":0},
```
Comment (noroadsleft) [inline layouts/default/65_ansi_blocker_tsangan/info.json:24]:
```suggestion
                {"x":13, "y":0, "w":2},
```

File: layouts/default/65_ansi_blocker_tsangan/layout.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+[{a:7},"","","","","","","","","","","","","","","",""],
```
Comment (noroadsleft) [inline layouts/default/65_ansi_blocker_tsangan/layout.json:1]:
```suggestion
[{a:7},"","","","","","","","","","","","","",{w:2},"",""],
```

File: layouts/default/65_ansi_blocker_tsangan/readme.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# 65_ansi_blocker_tsangan
+
+    LAYOUT_65_ansi_blocker_tsangan
+    
+This is a Tsanagan-inspired 65% ANSI layout with a blocker next to the arrows, split backspace, and 1.5u-1u-1.5u-7u-1.5u bottom row.
+
```
Comment (noroadsleft) [inline layouts/default/65_ansi_blocker_tsangan/readme.md:6]:
```suggestion
This is a Tsangan-inspired 65% ANSI layout with a blocker next to the arrows and 1.5u-1u-1.5u-7u-1.5u bottom row.
```

**Comment by brandonschlack** [comment]
Implemented feedback. Ready for review.

File: layouts/default/65_ansi_blocker_tsangan/default_65_ansi_blocker_tsangan/keymap.c [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+     * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┬───┐
+     * │ ` │ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │ 7 │ 8 │ 9 │ 0 │ - │ = │ Backsp│Hom│
+     * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┼───┤
+     * │ Tab │ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │ [ │ ] │  \  │PgU│
+     * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┼───┤
+     * │ Caps │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ; │ ' │  Enter │PgD│
+     * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────┬───┼───┤
+     * │ Shift  │ Z │ X │ C │ V │ B │ N │ M │ , │ . │ / │ Shift│ ↑ │End│
+     * ├─────┬──┴┬──┴──┬┴───┴───┴───┴───┴───┴───┴──┬┴───┴┬─┬───┼───┼───┤
+     * │Ctrl │GUI│Alt  │                           │  Alt│ │ ← │ ↓ │ → │
```
Comment (fauxpark) [inline layouts/default/65_ansi_blocker_tsangan/default_65_ansi_blocker_tsangan/keymap.c:14]: Could you make sure the spaces inside the box drawing here are NBSPs?

image

Comment (brandonschlack) [reply layouts/default/65_ansi_blocker_tsangan/default_65_ansi_blocker_tsangan/keymap.c]: Fixed the space characters in the preview.

File: layouts/default/65_ansi_blocker_tsangan/default_65_ansi_blocker_tsangan/keymap.c [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+     * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┬───┐
+     * │ ` │ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │ 7 │ 8 │ 9 │ 0 │ - │ = │ Backsp│Hom│
+     * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┼───┤
+     * │ Tab │ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │ [ │ ] │  \  │PgU│
+     * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┼───┤
+     * │ Caps │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ; │ ' │  Enter │PgD│
+     * ├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────┬───┼───┤
+     * │ Shift  │ Z │ X │ C │ V │ B │ N │ M │ , │ . │ / │ Shift│ ↑ │End│
+     * ├─────┬──┴┬──┴──┬┴───┴───┴───┴───┴───┴───┴──┬┴───┴┬─┬───┼───┼───┤
+     * │Ctrl │GUI│Alt  │                           │  Alt│ │ ← │ ↓ │ → │
```
Comment (fauxpark) [inline layouts/default/65_ansi_blocker_tsangan/default_65_ansi_blocker_tsangan/keymap.c:14]: Still two more in the Right Alt, sorry.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**